Docker.DotNet/IContainerOperations.cs

```diff
@@ -41,5 +41,9 @@ public interface IContainerOperations
         Task<Stream> GetContainerLogsAsync(string id, GetContainerLogsParameters parameters, CancellationToken cancellationToken);
 
         Task<Stream> CopyFromContainerAsync(string id, CopyFromContainerParameters parameters, CancellationToken cancellationToken);
+
+        Task<GetArchiveFromContainerResponse> GetArchiveFromContainerAsync(string id, GetArchiveFromContainerParameters parameters, bool statsOnly, CancellationToken cancellationToken);
```

Comment:
I think we should call it `stat` here, not `stats`. It refers to the stat system call in posix and not "statistics" which "stats" usually refers to. Also there is a stats `GET /containers/(id)/stats` endpoint, which is a different thing, but if we use "stats" in this method, it'll be confusing.